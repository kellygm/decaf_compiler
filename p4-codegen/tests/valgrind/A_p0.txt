==3517328== Memcheck, a memory error detector
==3517328== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==3517328== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==3517328== Command: ../decaf inputs/p0.decaf
==3517328== 
         #
        ###
       #####
      #######
     #########
    ###########
   #############
  ###############
 #################
###################
2
RETURN VALUE = 1
==3517328== 
==3517328== HEAP SUMMARY:
==3517328==     in use at exit: 0 bytes in 0 blocks
==3517328==   total heap usage: 5,363 allocs, 5,363 frees, 2,211,884 bytes allocated
==3517328== 
==3517328== All heap blocks were freed -- no leaks are possible
==3517328== 
==3517328== For lists of detected and suppressed errors, rerun with: -s
==3517328== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
