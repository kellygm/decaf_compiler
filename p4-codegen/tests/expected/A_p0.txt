         #
        ###
       #####
      #######
     #########
    ###########
   #############
  ###############
 #################
###################
2
RETURN VALUE = 1
